### DISCUSSION TITLE
Development

### DISCUSSION THREAD

**Discussion by VPpexis** [icpepsemapua/discordbot#8] [Announcements]
Hi guys!! I think we will start developing the Discord bot. Based from my experience in the discord server. One of the most critical bot "AppBot Legacy" tends to fail or offline. This caused a serious issues on processing new incomers to the bot.

With this we want to prioritize the development of a utility bot for role management, and user applications. Starting a new thread for the **development** discussion on the creation of the bot. We will also try to deploy it. We need first to discuss a time table of the development and the same time different requirements for the developments in the thread.

Thanks!!!

**Comment by VPpexis** [comment]
Possible commands:

- Configurable syntax command
- 'help' command
- 'review' command
- 'apply' command

**Reply by johnrivera0987** [reply]
- 'tambay' command - for viewing of voice channel stay duration
- 'announce' command
